analysers/Analyser_Osmosis.py

```diff
@@ -368,6 +396,12 @@ def requires_tables_build(self, tables):
                 elif table == 'touched_multipolygons':
                     self.requires_tables_build(['multipolygons'])
                     self.create_view_touched('multipolygons', 'R')
+                elif table == 'polygons':
+                    self.requires_tables_build(["multipolygons"])
+                    self.giscurs.execute(self.sql_create_polygons.format(self.config.db_schema.split(',')[0], self.config.options.get("proj")))
+                elif table == 'touched_polygons':
```

Comment:
Although not used yet, it may be useful to already add the `not_touched_polygons` table? It's not created anyway until it's requested by an analyser.